feature_engine/selection/probe_feature_selection.py

```diff
@@ -4,34 +4,23 @@
 import pandas as pd
 
 from feature_engine._docstrings.fit_attributes import (
-    _feature_names_in_docstring,
-    _n_features_in_docstring,
-)
+    _feature_names_in_docstring, _n_features_in_docstring)
```

Comment:
could we put this and the below back to the original representation? we use black for styling.

Reply:
same as above